### PR TITLE
Cutouts + light curves in anomaly notifications

### PR DESCRIPTION
**Pull request by Knispel2**
I have added 3 new functions:
* `get_cutout` - this function loads cutout image via Fink API;
* `get_curve` - this function loads light curve image via Fink API (I used the code from the example in Fink API documentation to build the curve);
* `get_data_permalink_slack` - using the two functions listed above, this function loads two images and copies these images to the Slack server.

I also put the status_code check of post requests into a separate function `status_check`, so as not to duplicate the code.

**_These versions of the modules are required to upload images to Slack_:
slack-sdk==3.21.3 or newer
slackclient==2.9.4 or newer**

Examples:
image
image

### CODE REVIEW

**Comment by Knispel2** [comment]
The code is ready for review

File: fink_filters/filter_anomaly_notification/filter_utils.py [hunk 1]
```diff
@@ -111,29 +241,111 @@ def get_OID(ra, dec):
...
+    for filt in np.unique(pdf['i:fid']):
+        maskFilt = pdf['i:fid'] == filt
+
+        # The column `d:tag` is used to check data type
+        maskValid = pdf['d:tag'] == 'valid'
+        plt.errorbar(
+            pdf[maskValid & maskFilt]['i:jd'].apply(lambda x: x - 2400000.5),
+            pdf[maskValid & maskFilt]['i:magpsf'],
+            pdf[maskValid & maskFilt]['i:sigmapsf'],
+            ls='', marker='o', color=colordic[filt]
```
Comment (JulienPeloton) [inline fink_filters/filter_anomaly_notification/filter_utils.py:326]: I would also add the `label=filt` argument

File: fink_filters/filter_anomaly_notification/filter_utils.py [hunk 2]
```diff
@@ -111,29 +241,111 @@ def get_OID(ra, dec):
...
+
+        maskBadquality = pdf['d:tag'] == 'badquality'
+        plt.errorbar(
+            pdf[maskBadquality & maskFilt]['i:jd'].apply(lambda x: x - 2400000.5),
+            pdf[maskBadquality & maskFilt]['i:magpsf'],
+            pdf[maskBadquality & maskFilt]['i:sigmapsf'],
+            ls='', marker='v', color=colordic[filt]
+        )
+
+    plt.gca().invert_yaxis()
```
Comment (JulienPeloton) [inline fink_filters/filter_anomaly_notification/filter_utils.py:344]: I would add the `plt.legend()` command to display the legend.

**Review by JulienPeloton — APPROVED** [review]
Thanks @Knispel2 ! For the tokens, we could use the Slack Fink_bot_test workspace, and the Telegram Fink_test channel? You can send me the token via email, and I will add them in the secrets of GitHub.

**Pull request merged**